code/__DEFINES/_bitfields.dm

```diff
@@ -1,4 +1,6 @@
-#define DEFINE_BITFIELD(_variable, _flags) /datum/bitfield/##_variable { \
+#define DEFINE_BITFIELD(_variable, _flags) \
+GLOBAL_REAL_VAR(_bitfield_##_variable); \
```

Comment:
you should comment that the intent of this is purely for compile time checks

Reply:
sure